components/apimgt/org.wso2.carbon.apimgt.gateway/src/test/java/org/wso2/carbon/apimgt/gateway/inbound/websocket/response/GraphQLResponseProcessorTest.java

```diff
@@ -190,4 +192,48 @@ public void testHandleInvalidScope() {
         Assert.assertEquals(processorResponseDTO.getErrorMessage(), graphQLProcessorResponseDTO.getErrorMessage());
         Assert.assertEquals(processorResponseDTO.getErrorCode(), graphQLProcessorResponseDTO.getErrorCode());
     }
+
+    @Test
+    public void testHandleResponseScopeValidationSkipWhenSecurityDisabled() {
+
+        InboundMessageContext inboundMessageContext = new InboundMessageContext();
+        int msgSize = 100;
+        String msgText = "{\"type\":\"data\",\"id\":\"1\",\"payload\":{\"data\":"
+                + "{\"liftStatusChange\":{\"name\":\"Astra Express\"}}}}";
+        PowerMockito.mockStatic(InboundWebsocketProcessorUtil.class);
+        InboundProcessorResponseDTO responseDTO = new InboundProcessorResponseDTO();
+        PowerMockito.when(InboundWebsocketProcessorUtil.authenticateToken(inboundMessageContext))
+                .thenReturn(responseDTO);
+
+        // VerbInfoDTO with security disabled
+        VerbInfoDTO verbInfoDTO = new VerbInfoDTO();
+        verbInfoDTO.setHttpVerb("SUBSCRIPTION");
+        verbInfoDTO.setThrottling("Unlimited");
+        verbInfoDTO.setAuthType("None");
+        GraphQLOperationDTO graphQLOperationDTO = new GraphQLOperationDTO(verbInfoDTO, "liftStatusChange");
+        inboundMessageContext.addVerbInfoForGraphQLMsgId("1", graphQLOperationDTO);
+
+        // Creating response for scope validation
```

Comment:
Same from here onwards as well. The successful response should be asserted as done in  **testHandleResponseSuccess**

Reply:
Same reason as mentioned above

Reply:
Ack. Thanks